Commands.sublime-commands

```diff
@@ -20,15 +20,20 @@
 		"args": { "paths": [] }
 	},
 	{
-		"caption": "File: Duplicate",
+		"caption": "File: Duplicate…",
 		"command": "side_bar_duplicate",
 		"args": { "paths": [] }
 	},
 	{
-		"caption": "File: Move",
+		"caption": "File: Move…",
 		"command": "side_bar_move",
 		"args": { "paths": [] }
 	},
+	{
+		"caption": "New…",
```

Comment:
Final thing: this needs a `File: ` prefix, because out of context, in the command palette, it's not clear what it will do.